openshift/metabase/oc_deploy_metabase.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -l
+#
+# Script based on instruction found in readme of
+# https://github.com/bcgov/nr-showcase-devops-tools/tree/master/tools/metabase
+
+export BASE_URL="https://raw.githubusercontent.com/bcgov/nr-showcase-devops-tools/master/tools/metabase/openshift"
+export ADMIN_EMAIL=BCWS.predictiveservices@gov.bc.ca
+export NAMESPACE=e1e498-dev
```

Comment:
I think these hardcoded namespaces should be parameterized so we can add metabase to our different environments.